cmd/keeper/keeper.go

```diff
@@ -1073,7 +1073,14 @@ func (p *PostgresKeeper) postgresKeeperSM(pctx context.Context) {
 				log.Errorw("failed to start instance", zap.Error(err))
 				return
 			}
-			// wait for the db having replyed all the wals
+
+			if standbySettings == nil {
```

Comment:
This check means that we'll waitRecoveryDone only when not syncing from an external instance (standby cluster). Is this really needed or it should be always done?

Reply:
I've never setup standby cluster, but I've guessed that recovery.done would never be created by PostgreSQL (since recovery should run forever until promoted to master).
If that guess is true, we should not wait here (or not with a timeout).

Also it seems desirable to be able to read from a standby cluster (e.g. for analytics) even if data is not the latest one.

Reply:
@PierreF You're right, I forgot that this is the pitr recovery case.